sessionctx/variable/sysvar.go

```diff
@@ -623,16 +630,25 @@ var defaultSysVars = []*SysVar{
 	{Scope: ScopeNone, Name: "ssl_ca", Value: ""},
 	{Scope: ScopeNone, Name: "ssl_cert", Value: ""},
 	{Scope: ScopeNone, Name: "ssl_key", Value: ""},
+	{Scope: ScopeGlobal, Name: InitConnect, Value: ""},
```

Comment:
This was moved from line 633 below because it incorrectly was in the section "tidb specific variables" (possibly from a strange merge). It is otherwise unrelated to this PR.